instance/tests/models/test_log_entry.py

```diff
@@ -39,6 +40,13 @@ class LogEntryTestCase(TestCase):
     """
     Test cases for LoggerInstanceMixin
     """
+    def test_default_log_level(self):
+        """
+        Check that the default log level is INFO
+        """
+        log_entry = GeneralLogEntry(text='OHAI')
+        self.assertEqual(log_entry.level, 'INFO')
```

Comment:
You could also relax this a bit to just check that the default value is a valid choice (but I'm fine with it either way).